src/pages/Calculator.tsx

```diff
@@ -98,11 +101,14 @@ export const Calculator = (): React.ReactElement => {
     addAlert('Link copied to clipboard!')
   }
 
+  const [pointsVisible, setPointsVisible] = useState(false)
```

Comment:
Is this used outside of the useMemo block below? It looks to me like `pointsVisible = points !== -1`

Reply:
removed